test/steps/browser.js

```diff
@@ -75,6 +94,16 @@ async function thenISee(expected: string): Promise<void> {
     await assert.textIsVisible(this.driver, expected);
 }
 
+async function thenIDontSee(expected: string): Promise<void> {
+    try {
+        await assert.textIsVisible(this.driver, expected);
```

Comment:
Although this might not work for the anti-case, because it thinks I'm looking for something, rather than not looking for it.